routes/billing.py

```diff
@@ -0,0 +1,63 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+from typing import List
+from .. import models, schemas
+from ..database import get_db
+
+router = APIRouter(prefix="/api/billing", tags=["Billing & Subscription"])
+
+
+#view
+@router.get("/", response_model=List[schemas.BillingOut])
+def get_subscriptions(db: Session = Depends(get_db)):
+    users = db.query(models.User).all()
+    return [
+        {
+            "id": u.id,
+            "user_id": u.id,
+            "user_name": f"{u.firstName} {u.lastName}",
+            "subscription_type": u.subscriptionType,
+            "subscription_status": u.subscriptionStatus,
+            "subscription_start": u.subscriptionStartDate,
+            "subscription_end": u.subscriptionEndDate,
+            "subscription_duration": u.subscriptionDurationType,
+        }
+        for u in users
+    ]
```

Comment:
intresting in db models these fields are not imported and in api you are returing these